string joint
int32 pwm_start
int32 pwm_end
int32 channel
int32 board